ci-operator/step-registry/openshift-qe/node-density-heavy/openshift-qe-node-density-heavy-ref.yaml

```diff
@@ -53,6 +53,26 @@ ref:
     default: "~/mno/kubeconfig"
     documentation: |-
       Set kubeconfig path inside the bm, default to ~/mno/kubeconfig  
+  - name: ADDITIONAL_PARAMS
+    default: ""
+    documentation: |-
+      JSON value to upload additonal metadata to ES index
+  - name: COMPARISON_CONFIG
```

Comment:
talked with Mehul offline, these parameters are used in an older version of kube-burner. Other than additional params I think they should be taken out